mailet/base/src/test/java/org/apache/mailet/base/test/FakeMail.java

```diff
@@ -259,7 +259,7 @@ public static FakeMail defaultFakeMail() throws MessagingException {
         return builder.build();
     }
 
-    private MimeMessage msg;
+    private transient MimeMessage msg;
```

Comment:
you serialize only the metadata?

Reply:
Yes... Mime message is stored via other means...